addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/converter/swagger/RequestModelConverter.java

```diff
@@ -88,4 +102,39 @@ private String generateBody() {
         }
         return "";
     }
+
+    private String extractExampleBody(Content content, String mediaTypeString) {
+        String exampleBody = null;
+
+        if (content.get(mediaTypeString).getExamples() != null
+                && !content.get(mediaTypeString).getExamples().isEmpty()
+                && content.get(mediaTypeString).getExamples().entrySet().stream()
+                                .findFirst()
+                                .get()
+                                .getValue()
+                        != null) {
+            Example example =
+                    content.get(mediaTypeString).getExamples().entrySet().stream()
+                            .findFirst()
+                            .get()
+                            .getValue();
+
+            if (example.getValue() != null) {
+                exampleBody = example.getValue().toString();
+            } else if (example.getExternalValue() != null
+                    && !example.getExternalValue().isEmpty()) {
+                Requestor requestor = new Requestor(HttpSender.MANUAL_REQUEST_INITIATOR);
```

Comment:
Not sure about doing these extra requests. In any case this is not correct when the definition is imported using the spider.